routes/routes_linux.go

```diff
@@ -81,6 +87,12 @@ func SetNetmakerPeerEndpointRoutes(defaultInterface string) error {
 	currentPeers := config.GetHostPeerList()
 	for i := range currentPeers {
 		peer := currentPeers[i]
+		if peer.Endpoint.IP == nil {
```

Comment:
this check will cause a nil pointer deference error if `peer.Endpoint is nil`

Reply:
changed back to if peer.Endpoint == nil